wear/build.gradle.kts

```diff
@@ -1,67 +1,17 @@
 plugins {
-    alias(libs.plugins.android.application)
-    alias(libs.plugins.kotlin.android)
-    alias(libs.plugins.ksp)
-    alias(libs.plugins.hilt)
+    alias(libs.plugins.homeassistant.android.application)
     alias(libs.plugins.google.services)
-    alias(libs.plugins.compose.compiler)
 }
 
 android {
-    namespace = "io.homeassistant.companion.android"
-
-    compileSdk = libs.versions.androidSdk.compile.get().toInt()
-
     defaultConfig {
         applicationId = "io.homeassistant.companion.android"
```

Comment:
Is this still needed? I think it is now defined in the application convention plugin.

Reply:
You're going to remove it